### ISSUE TITLE
Green tint in Gcam (primary lens) in view finder and clicked pictures.

### ISSUE THREAD

**Issue by bhardwaj-abhi** [ayush5harma/PixelFeatureDrops#48]
## Device Info

- Device - Poco X3 NFC ( Indian & Global both)
- AOSP - Dot OS 5.1
- Gapps enbaled.
- Android version -11

## Description

- After flashing this pixel prop ( june release) via magisk. 
 Green tint appears in view finder and in pictures also.
Usually tint appears in low light .

## Steps to reproduce

-flash this zip
-install Ng resource apk
- reboot.
- open gcam.  

## Expected Result

There should be no tint of any kind in any camera app, in this case Gcam.

##Actual Result

Green tint

![PXL_20210707_180756113](https://user-images.githubusercontent.com/85690594/125181556-80594980-e223-11eb-9c85-b6762c33bea7.jpg)

**Comment by ayush5harma** [comment]
Tint appears after flashing module? Which gcam are you using and did you try changing camera model for primary lens within your gcam settings?

**Comment by bhardwaj-abhi** [comment]
Yes, tint appears after flashing module. I've checked this issue in two different gcams (BSG, Nikita). Tried changing awb camera module with Gcam setting.  Tint issue still exist.
Flashing this zip change the gcam lens id for aux lens  too.

**Comment by ayush5harma** [comment]
> Yes, tint appears after flashing module. I've checked this issue in two different gcams (BSG, Nikita). Tried changing awb camera module with Gcam setting. Tint issue still exist.
> Flashing this zip change the gcam lens id for aux lens too.

Most gcams autoloads settings based on prop values so if your gcam is changing settings just take a backup as XML  of the gcam settings that worked for you  before flashing and restore it after the flash.

**Comment by bhardwaj-abhi** [comment]
issue is still there even after loading the xml again after flashing this zip.

**Comment by Orel6505** [comment]
Try updating your dotos version to 5.1.3

**Comment by L99996666** [comment]
The same Issue on POCO X3 NFC (surya), Arrow OS.
Greenish photos in GCAM.
Also GCAM crashes in Video mode
